src/api/app/models/patchinfo.rb

```diff
@@ -247,4 +257,10 @@ def issues_by_tracker
     end
     issues_by_tracker
   end
+
+  def description_length
+    unless description && description.length > [summary.length, 50].max
```

Comment:
I find this hard to understand and doubt it makes sense.

Reply:
@hennevogel Now we don't compare description length with summary length, only check if description is bigger than 50. Better now?